_sass/components/_glossary.scss

```diff
@@ -38,18 +39,22 @@ $term-width-max: 220px;
   display: inline-block;
   margin: 0;
   max-width: $term-width-max;
-  padding-bottom: $base-padding;
   width: 100%;
 
   @include respond-to(tiny-up) {
     width: $term-width;
+
+  @include heading('para-md');
+  font-weight: 700;
```

Comment:
This should definitely be indented #picky 
Also, the `font-weight` should be moved before the `width` and the `@include heading('para-md')` should be moved before both of them